dotcom-rendering/src/model/article-schema.json

```diff
@@ -3897,7 +3898,15 @@
                     }
                 },
                 "abTests": {
-                    "type": "object"
+                    "description": "This type is not support by JSON-schema, it evaluates as `object`",
```

Comment:
This might not be entirely accurate… It seems like the new type definition is actually more precise than just `object`.